docs/codelabs/codelab-chat-with-a-pdf.md

```diff
@@ -0,0 +1,206 @@
+# Chat with a PDF file
+
+This codelab shows you how to use Genkit to implement an app that lets you chat with a PDF file.
+
+## Prerequisites
+
+
+This codelab assumes that you’re familiar with building applications with Node.js. To complete this codelab, make sure that your development environment meets the following requirements:
+
+
+- Node.js v20+
+- npm
+
+## Create a new project
+
+- Create a new empty folder
+
+```shell
+mkdir chat-with-a-pdf
+cd chat-with-a-pdf
+```
+
+- Initialise a new TypeScript project
```

Comment:
Using UK spelling is my little subversive trait to protest the mind-boggling date format the US uses ;-)

But since all our docs use US spelling, I'm OK with updating it.

Reply:
Lolol. I cross my handwritten 7s and use 24-hour time format for its clarity, so I'm down with subversion. (But this is Git, not Subversion. sorry :-P )